### DISCUSSION TITLE
Add open badges to user profile on GitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#16290] [Other Feature Feedback, Questions, & Ideas]
Hi all! I would like to help Github in a positive way

#### 1. Problem
One of the current problems on the internet is to verify profiles... so to avoid fake profiles or boot

#### 2. Solution
1. There are several solutions to solve this, one solution initially is to say that a profile is verified, this happens with a "verified profile" icon or warning - this is not done for all users but only for user types, e.g. companies , communities, institutions and public figures - this serves to ensure reliability, integrity
2. Another way would be to assign certain open badges to the profiles... so it's possible to check if that user or that group of users is associated or is part of that open badge... What I'm requesting is an open badge as another form of verification in profile.

#### 3. What is the Open Badge?
"Open Badges is the world's leading format for digital badges. Open Badges is not a specific product or platform, but a type of digital badge that is verifiable, portable, and packed with information about skills and achievements" - [reference](https://openbadges.org/)

#### 4. user stories
1. As a user I would like to have an Open Badge associated with my user profile in case I am not a verified profile or if I want to have a second layer of verification.
2. I can have an Open badge associated with pull requests or donations and/or issues I open/close etc that I do.

#### 5. Notes
1. I believe that having 2 types of verification is interesting, so if the user cannot have a verified profile on Github, he has at least another way to be verified
2. Open Badges exist in several places, a good example of open badges as a reputation and scoring system is in the case of Mozilla Firefox - users have points associated with open badges, [here](https://connect.mozilla.org/t5/discussions/community-ranks-amp-badges/td-p/3744).
3. I don't want to promote Mozilla Firefox or the Open badges initiative, the examples I mentioned to references and bibliography
4. Currently Github offers an option to verify the profile, but not the option to open badges. So...  I would like to have open badges on Github. Also, open badge can work as a reputation and scoring system, which makes Github more secure and reliable. I'm not saying that Github is insecure, bad... so... I'm thinking of a way to make it better. 
5. If you are associated with an institution, company or your community, you get a profile icon - But it is not an open badge, it would be necessary to establish an open seal and set scores for that

#### 6. References
- https://connect.mozilla.org/t5/discussions/community-ranks-amp-badges/td-p/3744

**Comment by ghost** [comment]
#### Concept
![image](https://user-images.githubusercontent.com/95204402/173416514-28b7704d-5a8f-4e05-91af-9e02369b51ea.png)

**Comment by Eris-js** [comment]
reasonable
